akka-actor-typed-tests/src/test/java/akka/actor/typed/javadsl/StashBufferTest.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.actor.typed.javadsl;
+
+import akka.actor.testkit.typed.internal.StubbedActorContext;
+import akka.actor.testkit.typed.javadsl.LogCapturing;
+import akka.actor.typed.internal.StashBufferImpl;
+import org.junit.Rule;
+import org.junit.Test;
+import org.scalatestplus.junit.JUnitSuite;
+
+import java.util.function.Function;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class StashBufferTest extends JUnitSuite {
```

Comment:
This test is mainly to verify that the Java API is good enough when coding in Java. It does exercise some cases, but implementation wise, they are base on the Scala impl. So, not really needed.